src/js/tracks/text-track-list.js

```diff
@@ -111,6 +111,7 @@ TextTrackList.prototype.removeTrack_ = function(rtrack) {
   for (let i = 0, l = this.length; i < l; i++) {
     if (this[i] === rtrack) {
       track = this[i];
+      track.off();
```

Comment:
See the note in the doc for this method :)
The track in this list could be a native track element which won't have an `off` method. Can probably check `track instanceof TextTrack` or something.
However, because users are required to manage their listeners with native tracks, I wonder whether we should do this for the emulated tracks?